tsdb/head_test.go

```diff
@@ -500,7 +500,7 @@ func TestHead_ReadWAL(t *testing.T) {
 			s100 := head.series.getByID(100)
 
 			require.Equal(t, labels.FromStrings("a", "1"), s10.lset)
-			require.Equal(t, (*memSeries)(nil), s11) // Series without samples should be garbage collected at head.Init().
+			require.Equal(t, (*memSeries)(nil), s11) // Series without samples should be garbage collected at head.InitStepTracking().
```

Comment:
This seems incorrect to me.

Reply:
Doh, sorry for not catching that myself. That's IntelliJ's rename automation doing the very wrong thing.